CHANGELOG.md

```diff
@@ -8,6 +8,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Enhancements
 
+* Fix static_framework Swift pod dependencies and implement pod access to dependent vendored_framework modules  
```

Comment:
nit: probably place this in the `Bug Fixes` section can include #7117 link instead